sources/packages/web/src/views/aest/SearchInstitutions.vue

```diff
@@ -1,79 +1,82 @@
 <template>
   <full-page-container>
-    <h2 class="color-blue">Search Institution</h2>
-    <v-row class="mt-5">
-      <v-col>
-        <div class="p-fluid p-formgrid p-grid">
-          <div class="p-field p-col-12 p-md-4">
-            <label class="field-required" for="legalName">Legal Name</label>
-            <InputText
-              name="legalName"
-              data-cy="legalName"
-              v-model="legalName"
-              @keyup.enter="searchInstitutions"
-            />
-          </div>
-          <div class="p-field p-col-12 p-md-4">
-            <label class="field-required" for="operatingName"
-              >Operating Name</label
-            >
-            <InputText
-              name="operatingName"
-              data-cy="operatingName"
-              v-model="operatingName"
-              @keyup.enter="searchInstitutions"
-            />
-          </div></div
-      ></v-col>
-      <v-col class="mt-9" cols="auto"
-        ><v-btn
-          :disabled="!legalName && !operatingName"
-          color="primary"
-          data-cy="searchInstitutions"
-          @click="searchInstitutions"
-          >Search</v-btn
-        ></v-col
-      >
-    </v-row>
-
-    <DataTable
-      v-if="institutionsFound"
-      class="mt-4"
-      :autoLayout="true"
-      :value="institutions"
-    >
-      <Column field="legalName" header="Legal Name" :sortable="true">
-        <template #body="slotProps">
-          <div class="p-text-capitalize">
-            {{ slotProps.data.legalName }}
-          </div>
-        </template>
-      </Column>
-      <Column field="operatingName" header="Operating Name" :sortable="true">
-        <template #body="slotProps">
-          <div class="p-text-capitalize">
-            {{ slotProps.data.operatingName }}
-          </div>
-        </template>
-      </Column>
-      <Column field="address" header="Address">
-        <template #body="slotProps">
-          <div class="p-text-capitalize">
-            {{ getFormattedAddress(slotProps.data.address) }}
-          </div>
-        </template>
-      </Column>
-      <Column>
-        <template #body="slotProps">
-          <v-btn
+    <body-header title="Search Institution"> </body-header>
+    <content-group>
+      <v-row>
+        <v-col>
+          <v-text-field
+            density="compact"
+            label="Legal Name"
+            variant="outlined"
+            v-model="legalName"
+            data-cy="legalName"
+            @keyup.enter="searchInstitutions"
+            :rules="[(v) => !!v || 'Legal Name Required']"
```

Comment:
Is there a need to go all upper letters? Maybe just "Legal name required"

Reply:
Was it changed?

Reply:
oh..sorry. I missed it. pushing